### ISSUE TITLE
6.0 daily sha256sums include path

### ISSUE THREAD

**Issue by cassidyjames** [elementary/os#396]
These should just include the filename, but it includes a whole path. i.e. 
```
11178b35c3d03cbf86044a333dab743be130b12510edf5c1ee1305104c30e30f  /__w/os/os/builds/amd64/elementaryos-6.0-daily.20200812.iso
```

This prevents it from easily working with `sha256sum -c elementaryos-6.0-daily.20200812.sha256.txt`

**Comment by pongloongyeat** [comment]
Might be a dumb question but wouldn't a simple

``` bash
cd "$OUTPUT_DIR"
sha256sum "${FNAME}.iso" > "${FNAME}.sha256.txt"
cd "$BASE_DIR"
```

in

https://github.com/elementary/os/blob/3635fad8aa03438de3d28ae40d6cacb7aa54221a/build.sh#L92

work? It's a workaround and not a very clean solution but looking at `sha256sum --help`, there doesn't seem to be an option to disregard the relative path to `OUTPUT_DIR`.

**Comment by davidmhewitt** [comment]
@pongloongyeat Yes, that would work. Feel free to submit a PR with those changes.

**Comment by cassidyjames** [comment]
Fixed by #409
